arlas-commons/src/main/resources/roles.yaml

```diff
@@ -25,21 +25,21 @@ technicalRoles:
     - r:permissions:GET
 
   role/arlas/tagger:
-    - r:write/${org}.*:POST
-    - r:status/${org}.*:GET
+    - r:write/.*:POST
+    - r:status/.*:GET
 
   role/arlas/builder:
```

Comment:
These changes are not relative to the title of the commit and the PR.
Please add a specific commit